app/helpers/benchmarks_helper.rb

```diff
@@ -0,0 +1,63 @@
+module BenchmarksHelper
+  IDSEQ_BENCH_BUCKET = "idseq-bench".freeze
+  IDSEQ_BENCH_KEY = "config.json".freeze
+
+  def benchmarks_list
+    benchmarks = {}
+
+    # Read benchmarks from configuration store in S3
+    benchmark_config = benchmarks_config()
```

Comment:
this was a little confusing to read and I read it as `benchmark_config = benchmark_config` at first

Reply:
oh yeah... rubocop forced me to change the name and end up being the same. Renamed.